content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -0,0 +1,68 @@
+---
+title: How Application Security works in Datadog
+kind: guide
+---
+
+## Overview
+
+Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities.
+
+APM records information about each HTTP request, these are referred to as traces. Application security uses the information APM is already collecting, and flags any suspicious requests or attack attempts as security signals. Depending on your settings and the severity of the attempt, you can receive notifications from Slack, email, or PagerDuty.
+
+Traditional Web Application Firewalls (WAFs) are usually deployed at the perimeter and have no context of the application behavior. For Application Security to be effective, it’s critical to be embedded in the application to get access to the data. Application Security provides WAF-like capabilities with additional application context and lower false positives.
```

Comment:
> For Application Security to be effective
Confusing if about the domain or Datadog product. Maybe use lower-case and always refer to the product as "Datadog Application Security"?

> Application Security provides WAF-like capabilities with additional application context and lower false positives.
Application Security leverages known attack patterns like WAF and leverage the execution context to increase the signal/noise ratio, i.e reduce false positives